src/index.html

```diff
@@ -7,12 +7,66 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="product-card"
+      data-qa="card"
+    >
+      <div class="product-card__container">
+        <img
+          src="images/imac.jpeg"
+          alt="Mac Monoblock"
+          class="product-card__image"
+        />
+
+        <h2 class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="product-card__code">Product code: 195434</p>
+
+        <div class="product-card__review-container">
+          <ul class="product-card__review-stars stars stars--4">
```

Comment:
```suggestion
          <ul class="stars stars--4">
```
You don't use this class